scripts/functions/requirements/arch

```diff
@@ -40,9 +40,23 @@ requirements_arch_define_check_raspberry_pi()
   fi
 }
 
+requirements_arch_define_openssl()
+{
+  case "$1" in
+    (ruby-2.3*|ruby-2.2*|ruby-2.1*|ruby-2.0*|ruby-1.9*)
+      requirements_check openssl-1.0
+      __rvm_update_configure_openssl_dir_option "/usr/lib/openssl-1.0"
```

Comment:
I think the only path to fixing it might be:
```bash
export PKG_CONFIG_PATH="/usr/lib/openssl-1.0/pkgconfig${PKG_CONFIG_PATH:+:}${PKG_CONFIG_PATH:-}"
```
[I checked with ruby - it should respect `pkg-config` in case of `openssl`](https://github.com/ruby/ruby/blob/trunk/ext/openssl/extconf.rb#L38) (it does not have to be the case with other extensions).